### PR TITLE
input filter for list with items

### PR DESCRIPTION
**Pull request by moshesrur**

### CODE REVIEW

File: react/filter-items-from-list/src/filter.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Filter(props) {
```
Comment (ilyador) [inline react/filter-items-from-list/src/filter.js]: You can do the destructure directly in the function definition:
```js
export default function Filter({filter, setFilter}) {
```

File: react/filter-items-from-list/src/itemslist.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export default function ItemsList(props) {
+  const {filter} = props;
+  const cars = ["alfa", "audi", "bmw", "cadillac", "dodge", "opel", "seat",
+    "saab", "skoda", "fiat", "volvo", "volkswagen"];
+
+  return (
+      <div>
+        <select id="cars" name="cars" size={10}>
+          {cars.map((car, index) => {
+            return car.includes(filter) && <option key={index} value={car}>{car}</option>
```
Comment (ilyador) [inline react/filter-items-from-list/src/itemslist.js:12]: Nice solution:)

File: react/filter-items-from-list/src/main.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Filteredlist from "./filteredlist";
+
+const App = () => {
+  return (
```
Comment (ilyador) [inline react/filter-items-from-list/src/main.js:6]: No need to wrap a single element with a fragment, you can simply do `return <Filteredlist />`

**Review by ilyador — COMMENTED** [review]
Good work

File: react/filter-items-from-list/src/main.js (hunk 3 above)
Comment (moshesrur) [reply react/filter-items-from-list/src/main.js:6]: done

File: react/filter-items-from-list/src/filter.js (hunk 1 above)
Comment (moshesrur) [reply react/filter-items-from-list/src/filter.js]: done

**Pull request merged**